ui/src/api/logs/schema.ts

```diff
@@ -1,31 +1,74 @@
-import { LogLevelSchema } from "../schema";
 import { z } from "zod";
 
-const LogEntrySchema = z.object({
-  t: z.string(), // "2023-07-27T09:49:58.408869Z"
-  level: LogLevelSchema,
-  msg: z.string(), // "Preparing workflow triggered by api."
-  tags: z.object({
-    callpath: z.string(), // "/"
-    "instance-id": z.string(), // "4b833cfd-a0ef-4f4c-994b-4b63c7301778"
-    invoker: z.string(), // "api", "cron", "cloudevent", "complete" (if it's created as a subflow from another instance it's something like instance:%v, where %v is the instance ID of its parent
-    "loop-index": z.string().optional(), // ""
-    namespace: z.string(), // "workflow"
-    "namespace-id": z.string(), // "d1df8d21-96ce-49eb-98b7-5af7571ab28f"
-    recipientType: z.string(), // "instance"
-    workflow: z.string(), // "workflow.yaml"
-    "workflow-id": z.string().optional(), // "a3acd2eb-75de-4369-bb1e-41d5364ea6b7"
-    "state-id": z.string().optional(), // "a3acd2eb-75de-4369-bb1e-41d5364ea6b7"
-  }),
+export const LogLevelSchema = z.enum(["INFO", "ERROR", "WARN", "DEBUG"]);
+export type LogLevelSchemaType = z.infer<typeof LogLevelSchema>;
+
+/**
+ * example
+ * 
+{
+  "status": "completed",
+  "state": "loop",
+  "branch": null,
+  "workflow": "/wf.yaml",
+  "calledAs": null,
+  "instance": "f1242c40-3cd9-48bb-82aa-02275df6e1da" 
+}
+ */
+export const WorkflowLogData = z.object({
```

Comment:
I'm not sure about that. The idea of the structure is, that there is a common set of log data that every type of log entry has (that data is placed at the root level). And depending on what generated the log, there might be additional log information that are nested in a special attribute to avoid "polluting the root layer". There are even more type than just `workflow`, but I did not add them to the schema because we don't use them (yet).

Reply:
I understand, but that's exactly why I find it confusing to have "WorkflowLogData" as a child of a single entry, because "Log" for me always suggests a list of entries, not a subset of data within one entry. So something like "WorkflowStatusData" or "WorkflowData" would make more sense for me.

Reply:
Okay, now I got it. You are right. I wrote the comment before I read your naming suggestions about logs vs log lines. Now this makes much more sense to me (2c7e4172e185c26d4799d3141781b09005381104)